docs/Shell-Completion.md

```diff
@@ -8,7 +8,7 @@ You must configure your shell to enable its completion support. This is because
 
 ## Configuring Completions in `bash`
 
-To make Homebrew's completions available in `bash`, you must source the definitions as part of your shell's startup. Add the following to your `~/.profile` file:
+To make Homebrew's completions available in `bash`, you must source the definitions as part of your shell's startup. Add the following to your `~/.profile` or `~/.bash_profile` file:
```

Comment:
```suggestion
To make Homebrew's completions available in `bash`, you must source the definitions as part of your shell's startup. Add the following to your `~/.bash_profile` if it exists or `~/.profile` otherwise:
```

Reply:
Done!